include/ipmitool/helper.h

```diff
@@ -183,11 +189,23 @@ uint8_t *array_letoh(uint8_t *buffer, size_t length);
 #endif
 
 #ifndef __minlen
-# define __minlen(a, b) ({ int x=strlen(a); int y=strlen(b); (x < y) ? x : y;})
+static inline size_t min_strlen(const char *a, const char *b)
```

Comment:
Yup. Already have this done in my local branch.

Reply:
Ok, I will remove this.